### DISCUSSION TITLE
How to make all Biome lint errors warnings instead of errors

### DISCUSSION THREAD

**Discussion by AVGVSTVS96** [biomejs/biome#3634] [Q&A]
I want to be able to differentiate between real errors and linting errors at a glance. I know theres a way to do this in eslint, is there any way in Biome, without manually defining every rule as warning in the configuration? I think this is amazing essential feature for a linter.

**Comment by ematipico** [comment]
That isn't possible nowadays 

Duplicate of https://github.com/biomejs/biome/discussions/3106
